repository/sql_option_test.go

```diff
@@ -160,20 +160,20 @@ func (s *SQLOptionsTestSuite) TestFindOffsetOption() {
 	s.Assert().EqualValues([]int{1, 2, 3, 4, 5, 6, 7, 8, 9, 10}, s.getValues(out))
 }
 
-func (s *SQLOptionsTestSuite) TestFindSelectAndGroupByOptions() {
-	var out []*SQLOptionsTestModel
-
-	// GroupBy fails if unaggregated fields are returned
-	s.Assert().Error(s.repository.Find(&out, GroupBy("even")))
-
-	// Single field group
-	s.Assert().NoError(s.repository.Find(&out, Fields("SUM(value) value"), GroupBy("even")))
-	s.Assert().EqualValues([]int{1 + 3 + 5 + 7 + 9, 2 + 4 + 6 + 8 + 10}, s.getValues(out))
-
-	// Multiple field group
-	s.Assert().NoError(s.repository.Find(&out, Fields("SUM(value) value"), GroupBy("even", "lte5")))
-	s.Assert().EqualValues([]int{1 + 3 + 5, 2 + 4, 6 + 8 + 10, 7 + 9}, s.getValues(out))
-}
+//func (s *SQLOptionsTestSuite) TestFindSelectAndGroupByOptions() {
+//	var out []*SQLOptionsTestModel
+//
+//	// GroupBy fails if unaggregated fields are returned
+//	s.Assert().Error(s.repository.Find(&out, GroupBy("even")))
+//
+//	// Single field group
+//	s.Assert().NoError(s.repository.Find(&out, Fields("SUM(value) value"), GroupBy("even")))
+//	s.Assert().EqualValues([]int{1 + 3 + 5 + 7 + 9, 2 + 4 + 6 + 8 + 10}, s.getValues(out))
+//
+//	// Multiple field group
+//	s.Assert().NoError(s.repository.Find(&out, Fields("SUM(value) value"), GroupBy("even", "lte5")))
+//	s.Assert().EqualValues([]int{1 + 3 + 5, 2 + 4, 6 + 8 + 10, 7 + 9}, s.getValues(out))
+//}
```

Comment:
For some reason, different test executions led to different results when checking the value order. I don't think it makes sense to fix this in this PR, but in a future one instead.

Reply:
I imagine this is happening because the database is not being properly cleaned from running other tests before running this one.

+1 to fixing this in another PR.

Reply:
I added an issue to this PR: #9